props/saber_BC_buttons.h

```diff
@@ -1253,9 +1270,13 @@ struct BCVolumeMode : public SPEC::SteppedMode {
       if (current_volume_ >= max_volume_) {
         current_volume_ = max_volume_;
         QuickMaxVolume();
-      } else {
+      } else if (millis() - trigger_time_ > 100 && !sound_library_.busy()) {
```

Comment:
Why do you need to check both busy() and trigger_time_ ?
Seems like a suspenders-and-belt solution?

Reply:
Trigger Time 100 typically ends up being longer than the actual sound length of the small click sound (that's good). Because I wanted to space those click sounds out but also handle if a longer prompt such as maximum volume was spoken.

Reply:
I don't see how this spaces out click sounds?

Reply:
Because if the click sound is only 20 milliseconds, too many queue up with fast rotation. This is limiting calls to sound library to a minimum of 100 milliseconds.

Reply:
but how? There is no click playing in here

Reply:
Wait, I think I'm confused "SayVolumeDown" is the tick, isn't it?
I was thinking that this was the min/max case, let me take another look at the code.

Reply:
I'm not a big fan of hard-coding time limits.
But I guess it's ok since there is also a busy() check in there.

It's not what I would have done though. This just drops the sound, which means that users can end up changing the volume several steps but only get one sound.  I would have just called fadeout(0.2) to get rid of the previous sound and then played the new sound instead.  However, if you are happy with how this works, that's fine with me.

Reply:
Calling fadeout is probably the right and more simple way to handle most of this. I guess I couldn't figure out how to target the sound that's playing? I'll have to look again.

I will play around when I get back.